src/zen/tlsmanager.cpp

```diff
@@ -12,6 +12,8 @@
 #include "tlsmanager.h"
 #include "utiltls.h"
 
+extern std::unique_ptr<CConnman> connman;
```

Comment:
Yes, I totally agree with you!
Bitcoin groups all the pointers to modules in the global structure `NodeContext` (see src/node/context.h in Bitcoin repo). The long term plan for us is to replicate this model and go on with the modularization of most of zend subsystems, but at the moment only the connection manager and the mempool are ready for this.
A possible plan for the short term is to create the NodeContext and add these two components in a future PR, as this could be done in a separate and independent task; at the moment, however, we should endure with globals...